addon/templates/components/frost-text.hbs

```diff
@@ -0,0 +1,4 @@
+{{input type="text" autofocus=autofocus disabled=disabled value=value placeholder=placeholder readonly=readonly}}
```

Comment:
Please line-wrap the options - we'll add template linting to cover this at a later stage

{{input
  type='text'
  autofocus=autofocus
  ...